fastlane/lib/fastlane/actions/push_to_git_remote.rb

```diff
@@ -52,6 +55,11 @@ def self.available_options
                                        description: "Force push to remote",
                                        is_string: false,
                                        default_value: false),
+          FastlaneCore::ConfigItem.new(key: :force_with_lease,
+                                       env_name: "FL_PUSH_GIT_FORCE_WITH_LEASE",
+                                       description: "Force push with lease to remote",
+                                       is_string: false,
```

Comment:
Can we change this line to `type: Boolean` instead? 😊

Reply:
And while you are in there, can you change the other `is_string: false` in the other two options to `type: Boolean` as well? 😊

Reply:
`type: Boolean` is preferred so the `Fastlane.swift` knows what type of arguments are needed